scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -785,6 +795,9 @@ private void processAlertRecord(final Record record, final Connection connection
 					errorHandler.onError(alert.getPeer(), alert.getLevel(), alert.getDescription());
```

Comment:
There are tests based on AlertMessages. To change them, givenAnEstablishedSession must be extended. Currently I detected some (very small) race conditions in the flight cancel. So I think elimination of the ErrorHandler should be shifted ;-).